db/user_list.sql

```diff
@@ -0,0 +1,8 @@
+-- movies.user_rating definition
+
+CREATE TABLE `user_rating` (
+  `user_id` int(11) UNSIGNED NOT NULL,
+  `movie_id` int(32) UNSIGNED NOT NULL,
+  `movie_rating` DECIMAL(3, 1) NOT NULL,
```

Comment:
would we perhaps like to allow NULL for move_ratings since this could technically be an unspecified value?

Reply:
I think you got the tables swapped based on the script names. As for the rating, since this table is specifically meant for users to rate a movie, not necessarily add them to their list, the rating should always have a value. It might help to try and imagine the workflow:

1. User searches a genre
2. Search returns movies A, B, and C
3. User clicks on movie B
4. A new page opens with movie B and all its info
5. User can click a + button to add it to their list, or user can assign a rating by clicking the number of stars they want

Does that help?

Reply:
yeah - I think my most recent commit address some of your comments too